### PR TITLE
Fix bug when starting evaluation devolution

### PR DESCRIPTION
**Pull request by Abriscioli**
@andresmoschini 

A minor change to reload the page and get the updated information in evaluation dashboard

### CODE REVIEW

File: source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js [hunk 1]
```diff
@@ -268,7 +268,7 @@
                 contentType: 'application/json; charset=utf-8',
                 data: JSON.stringify({ evaluationId: this.id }),
                 success: function (response) {
-                    getDashboardEvaluations(window.ViewData.period);
+                    window.location.reload();
```
Comment (andresmoschini) [inline source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js]: Does it not give us problem in any other scenario?

**Comment by andresmoschini** [comment]
:+1:

File: source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js (hunk 1 above)
Comment (Abriscioli) [inline source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js]: Not that I can recall :confused: 
Do you see any possible scenario that may be a problem?

Comment (andresmoschini) [inline source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js]: No, no, maybe something like the user lost something done at client side, but I have no idea.

Comment (Abriscioli) [inline source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js]: he only thing that comes to my mind is that the user tries to add or remove evaluators, and without accepting changes, starts the evaluation devolution. in that case, any change done to the evaluators list will be lost, but it's kind of an illogical action :confused:

**Pull request merged**